### DISCUSSION TITLE
packageRules and package version selection

### DISCUSSION THREAD

**Discussion by karlismelderis-mckinsey** [renovatebot/renovate#19629] [Request Help] [answered]
Hi,

I'm trying to understand limitations of creating package groups with `packageRules`.
All examples I find are grouping packages from same "scope" hence packages are releases in one bundle and share same version.

Would it be possible to group packages that has different version track?

```json
    {
      "matchPackageNames": ["packageA", "packageB"],
      "groupName": "same-topic packages"
    },
```
e.g. 
* packageA => 1.2.3
* packageB => 2.3.4

and have both updated in same PR?

**Comment by viceice** [comment, marked as answer]
yes, but the PR is then immortal and will be recreated if you close it 😉

**Reply by karlismelderis-mckinsey** [reply]
Good catch regards closed PRs.

It would probably work for my case.
